pkgs/shells/zsh/pure-prompt/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "pure-prompt-${version}";
```

Why did you name the package `pure-prompt`? The original project is simply called `pure`.
Also you can use `pname` instead of `name`, this will automatically append the version.

```suggestion
  pname = "pure";
```

---

Ah I see, there is already a package called `pure`. Updated suggested change:

```suggestion
  pname = "pure-prompt";
```

---

Amended: name -> pname.